packages/beacon-node/test/utils/state.ts

```diff
@@ -127,10 +141,14 @@ export function generateCachedAltairState(opts?: TestBeaconState, altairForkEpoc
 export function generateCachedBellatrixState(opts?: TestBeaconState): CachedBeaconStateBellatrix {
   const config = getConfig(ForkName.bellatrix);
   const state = generateState(opts, config);
-  return createCachedBeaconState(state as BeaconStateBellatrix, {
-    config: createBeaconConfig(config, state.genesisValidatorsRoot),
-    // This is a performance test, there's no need to have a global shared cache of keys
-    pubkey2index: new PubkeyIndexMap(),
-    index2pubkey: [],
-  });
+  return createCachedBeaconState(
+    state as BeaconStateBellatrix,
+    {
+      config: createBeaconConfig(config, state.genesisValidatorsRoot),
+      // This is a performance test, there's no need to have a global shared cache of keys
+      pubkey2index: new PubkeyIndexMap(),
+      index2pubkey: [],
```

Comment:
should rename this properties into "finalizedPubkey2index" and "finalizedIndex2pubkey"